src/ifp/README.md

```diff
@@ -9,13 +9,15 @@ initialize_floorplan
   [-site site_name]               LEF site name for ROWS
   -die_area "lx ly ux uy"         die area in microns
   [-core_area "lx ly ux uy"]      core area in microns
+  [-site site_names]              list of sites to make rows for
```

Comment:
-site now occurs twice - it isn't really part of the 'or' and was positioned first before..  If it takes a list it should be -sites